### PR TITLE
28 custum error wrapper

### PR DESCRIPTION
**Pull request by Saeed99Madi**

### CODE REVIEW

**Review by AseelL — APPROVED** [review]

**Review by mohmmed23 — DISMISSED** [review]

File: src/helpers/errorsHandling/CustumError.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CustumError extends Error {
+  status:number
+  message:string
+
+  constructor (status:number, message:string) {
+    super(message)
+    this.status = status
+    this.message = message
```
Comment (aaamra) [inline src/helpers/errorsHandling/CustumError.ts:8]: i think this line is not necessary because line 6 binds the message

**Review by aaamra — CHANGES_REQUESTED** [review]
great work

but i want from you to add the express error middleware and send a response with the error message and the status from it

File: src/helpers/errorsHandling/CustumError.ts (hunk 1 above)
Comment (Saeed99Madi) [reply src/helpers/errorsHandling/CustumError.ts:8]: do you mean line 8 ? @aaamra

Comment (aaamra) [reply src/helpers/errorsHandling/CustumError.ts:8]: yes @Saeed99Madi

**Review by salsabeelomar — APPROVED** [review]

**Review by mohmmed23 — APPROVED** [review]

**Pull request merged**